### PR TITLE
Add Content Warning to stories

### PR DESCRIPTION
**Pull request by silent1mezzo**

### CODE REVIEW

**Comment by davesteinberg** [comment]
The latest design has the content warning label replacing the category label in the story cards on the feed, in order to keep the size constant (Figma is up to date). Also, there's an existing `Label` common component that you can reuse for it, you just need to pass in `color` and `background` props (it automatically uses `color` for `borderColor`, as well).

File: components/common/Warnings.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Box } from '@chakra-ui/react'
+
+export function ContentWarningBox({ ...props }) {
```
Comment (JamiesonRoberts) [inline components/common/Warnings.tsx:3]: Type on this please! BoxProps should be the correct one

**Review by JamiesonRoberts — CHANGES_REQUESTED** [review]

**Review by mwickett — APPROVED** [review]
Once that type issue is resolved, LGTM.

**Review by JamiesonRoberts — APPROVED** [review]

**Comment by JamiesonRoberts** [comment]
LGTM! 🚀

**Pull request merged**